README.md

```diff
@@ -99,32 +99,40 @@ ansible-playbook -i hosts -v deploy-okd.yml --extra-vars "compliance_operator=tr
 Once the playbook tell you to, boot the masters on
 Fedora CoreOS USB and start the installation process. 
 The NUCs are a bit slow on the network side so a number of kernel arguments are needed.
+
+```shell
+# Automated version
+$ bash -c "$(curl -fsSL http://infra1.okd4.example.com:8080/install.sh)"
+```
+
 ```shell
+# Manual version
```

Comment:
Please harmonize this with the line above, for example:

# Manually set hostname
$ bash -c "$(curl -fsSL http://infra1.okd4.example.com:8080/install.sh)" -s master[1-3]